packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -157,6 +157,38 @@ function FlowsheetDataGrid(props) {
     isResizable: false,
   })), [columns]);
 
+  const displayedColumnsWithColumnSpan = useMemo(() => {
+    let i = 0;
+    const newDisplayedColumns = [];
+    columns.forEach((column) => {
+      if (column.columnSpan > 1) {
+        newDisplayedColumns[i] = {
+          ...column,
+          columnSpanIndex: 0,
+          isSelectable: false,
+          isSelected: false,
+          isResizable: false,
```

Comment:
Why ware we changing these property values?

Reply:
These values are required to be added to Column Object for things to work properly. Similar to how we have [here](https://github.com/cerner/terra-framework/blob/main/packages/terra-data-grid/src/FlowsheetDataGrid.jsx#L156). 
But this has been updated to use the flowsheetColumns to create the new object.